src/pages/embed.tsx

```diff
@@ -0,0 +1,21 @@
+import { NextPageContext } from 'next';
+import App from '../App';
+import { withoutEmbedQueryParams, parseEmbedQuery } from '../utils';
+
+export default function Embed({ query }: { query: any }) {
+  const embed = parseEmbedQuery(query);
+  return (
+    <App
+      embed={{
+        ...embed,
+        isEmbedded: true,
+        embedUrl: withoutEmbedQueryParams(query),
+      }}
+    />
+  );
+}
+
+Embed.getInitialProps = async (ctx: NextPageContext) => {
```

Comment:
you might have wanted [`getStaticProps`](https://nextjs.org/docs/basic-features/data-fetching#getstaticprops-static-generation) - `getInitialProps` is sort of deprecated/fading away

Reply:
fixed in the latest changes.